cli/waiter/subcommands/signal.py

```diff
@@ -0,0 +1,57 @@
+
+from waiter.util import guard_no_cluster, check_positive
+from waiter.action import process_signal_request
+from waiter.instance_select import get_instance_id_from_destination, Destination
+from enum import Enum
+
+
+class Signal(Enum):
+    SIGKILL = 'sigkill'
+    SIGTERM = 'sigterm'
+    HARD_DELETE = 'hard-delete'
+    SOFT_DELETE = 'soft-delete'
```

Comment:
SIGKILL and SIGTERM can be removed, hard-delete and soft-delete are aliases to those commands.